bigbluebutton-html5/imports/api/users/server/modifiers/addUser.js

```diff
@@ -40,6 +45,7 @@ export default function addUser(meetingId, user) {
       'user.userid': userId,
       'user.extern_userid': user.extern_userid,
       'user.role': user.role,
+      'user.roles': userRoles,
```

Comment:
Have you considered checking for roles somewhere else - I am not convinced that we need to store this in the collection